### ISSUE TITLE
trailing closure indent changed from 0.48.4 to 0.48.6

### ISSUE THREAD

**Issue by Cyberbeni** [nicklockwood/SwiftFormat#965]
```
let alert = UIAlertController.tr_alert(title: title, message: nil, otherButtonTitle: okTitle, otherButtonCallback: {
    okBlock()
}, cancelButtonTitle: cancelTitle) {
    cancelBlock()
}
```
`cancelBlock()` gets indented once more when formatting this with 0.48.6

**Comment by Cyberbeni** [comment]
(Not all of the trailing closures got an extra indent, I assume that the tests already contain examples that kept their original formatting)

**Comment by nicklockwood** [comment]
@Cyberbeni fixed in 0.48.7
